README.md

```diff
@@ -122,6 +124,18 @@ Setup Environment, by initiating the build:
 
 Here `yarn`: Install deps, create symlinks, hoist packages. & `yarn build:all`: Build all packages.
 
+To run RabbitMQ with the management plugin using Docker, open a new terminal window and run the following command:
```

Comment:
We don't need to say open a new terminal.
Here and in other places

Reply:
Sure ✔️